lib/jekyll-import/importers/easyblog.rb

```diff
@@ -67,8 +67,8 @@ def self.process(options)
           slug = post[:alias]
           date = post[:created]
           content = post[:content]
-          category = post[:category]
-          tags = post[:tags]
+          # category = post[:category]
```

Comment:
Same comment about comments here. :smiley:

Reply:
are these lines no longer being used? if so, why not?